dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/PIFreeze.mo

```diff
@@ -0,0 +1,59 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block PIFreeze "Proportional-integrator controller with freezing of the state"
+  import Modelica;
+
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Types.PerUnit Y0 = 0 "Initial value of output" annotation(
```

Comment:
Should be put just before the equation section as it is an init parameter, thanks

Reply:
thx!